capstone/src/main/java/com/google/step/clients/MetricsClient.java

```diff
@@ -0,0 +1,203 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.step.clients;
+
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.FetchOptions;
+import com.google.appengine.api.datastore.Query.FilterPredicate;
+import com.google.appengine.api.datastore.Query.CompositeFilter;
+import com.google.appengine.api.datastore.Transaction;
+
+import com.google.step.data.RestaurantPageViews;
+import com.google.step.data.WeeklyPageView;
+
+import java.util.Map;
+import java.util.HashMap;
+import java.util.List;
+import java.util.ArrayList;
+import java.util.Calendar;
+import java.util.Arrays;
+
+/**
+ * This client is in charge of interacting with metricsServlet and the metrics entities.
+ */
+public class MetricsClient {
+
+    /**
+     * Empty constructor since we don't need any variables, we only use this class for the methods.
+     */
+    public MetricsClient() {}
+
+    private static DatastoreService dataStore = DatastoreServiceFactory.getDatastoreService();
+
+    /**
+     * Gets the current week's page views for one restaurant by specifying a restaurantKey.
+     * @param restaurantKey restaurant key of the restaurant we want to get views.
+     * @param restaurantName restaurant name in case no entity exists for this week's page views.
```

Comment:
Nit: Order of params in java doc should match order of params in the function signature.

Reply:
Corrected.